Anlab.Mvc/Controllers/OrderController.cs

```diff
@@ -39,8 +42,13 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
         public async Task<IActionResult> Create()
         {
             var joined = await  _orderService.PopulateTestItemModel();
+            var proc = await _labworksService.GetPrice("PROC");

```

Comment:
@laholstege move this into a const or an app setting

Reply:
I'd say Const.